Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/ModelManager.java

```diff
@@ -42,6 +55,23 @@ public ModelInfoLoader getModelInfoLoader() {
         return this.modelInfoLoader;
     }
 
+    public boolean isDefaultModelInfoLoadingEnabled() {
+        return enableDefaultModelInfoLoading;
+    }
+
+    public void setEnableDefaultModelInfoLoading(boolean enableDefaultModelInfoLoading) {
+        this.enableDefaultModelInfoLoading = enableDefaultModelInfoLoading;
+    }
+
+    public void disableDefaultModelInfoLoading() {
+        this.enableDefaultModelInfoLoading = false;
```

Comment:
This would never have any effect, since the option is used in the initialize. For that reason, it's not clear that it should be a settable option.

Reply:
Removed the setter.
Also Added an Translator option `DisableDefaultModelInfoLoad` . 
Added a command line argument.
The only use case I found Translator options can dictate modelManager creation is in CqlTranslator main() method.
Thanks!